### DISCUSSION TITLE
Is there a way to force the spotlight to update without resizing/moving the mouse?

### DISCUSSION THREAD

**Discussion by wendtcode** [gilbarbara/react-joyride#630] [Q&A] [answered]
I'm spotlighting some content that is clamped at 2 visible lines, with a "more" toggle to view the additional content. When the content is expanded, the spotlight does not resize _until I move the mouse_. Is there a way to force the spotlight to resize _upon clicking the "more" toggle_?

https://user-images.githubusercontent.com/48448749/104374100-fb48de00-54e6-11eb-9b48-a77228afc840.mp4

**Comment by gilbarbara** [comment, marked as answer]
Did you try to dispatch a window resize event?

**Comment by wendtcode** [comment]
Nice! I hadn't thought to just manually trigger the event. Appreciate the help @gilbarbara!

**Comment by awwwdev** [comment]
For those who don't know "dispatch a window resize event" mean, just add the line below to the onClick function of the button that changes the size of content. 

window.dispatchEvent(new Event('resize'));

**Comment by krehwell** [comment]
this is a good question that I think they must documented it
